parser-typechecker/src/Unison/TermParser.hs

```diff
@@ -1,7 +1,6 @@
 {-# OPTIONS_GHC -Wno-partial-type-signatures #-}
 
-{-# LANGUAGE ViewPatterns #-}
-{-# LANGUAGE RankNTypes #-}
```

Comment:
Accidentally lost linebreak?

Reply:
No it wasn't intentional. Thx for spotting

Reply:
the LANGUAGE options shouldn't change